var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -462,6 +463,7 @@ def define_enable(suffix, value=None):
             define_trilinos_enable('ALL_OPTIONAL_PACKAGES', False),
             define_trilinos_enable('ALL_PACKAGES', False),
             define_trilinos_enable('CXX11', True),
+            define('Trilinos_CXX11_FLAGS', ' '),
```

Comment:
Can you put this in a conditional block below so that it only applies to Trilinos 12?

```python
if spec.version < Version('13'):
    # Suppress TriBITS flags in favor of CMake's built-in flags
    options.append(define('Trilinos_CXX11_FLAGS', ' '))
```

Reply:
Definitely agree with the conditional block. This isn't an issue with Trilinos injecting `-std=c++11` for versions `>=13`

Also, though, this solution actually causes build issues:

```
1 error found in build log:
     369    -- Performing Test CXX11_CONSECUTIVE_RIGHT_ANGLE_BRACKETS
     370    -- Performing Test CXX11_CONSECUTIVE_RIGHT_ANGLE_BRACKETS - Failed
     371    -- Performing Test CXX11_AUTOTYPEDVARIABLES
     372    -- Performing Test CXX11_AUTOTYPEDVARIABLES - Success
     373    -- Performing Test CXX11_LAMBDAS
     374    -- Performing Test CXX11_LAMBDAS - Failed
  >> 375    CMake Error at cmake/tribits/core/package_arch/TribitsGlobalMacros.cmake:1879 (MESSAGE):
     376      Error, C++11 support does not appear to be supported with this C++ compiler
     377      and/or with the C++11 flags Trilinos_CXX11_FLAGS=' '! If the flags
     378      Trilinos_CXX11_FLAGS=' ' where set manually, then try clearing the CMake
     379      cache and configure without setting Trilinos_CXX11_FLAGS and let the
     380      configure process try to find flags that work automatically.  However, if
     381      these compile-time tests still fail, consider selecting a different C++
```